webrob/templates/pages/QA.html

```diff
@@ -8,6 +8,8 @@
 
   <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/SCSS/split.css') }}"/>
   
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
This should be unneeded. We depend on the bootstrap node_module, and bootstrap CSS should be loaded on the QA page already.
It is further not ok to do imports from http source, openEASE needs to run offline without relying on browser cache or so.